scheduler/plugin/plugin.go

```diff
@@ -43,7 +42,12 @@ func New(configuration runtime.Object, frameworkHandle framework.Handle) (framew
 
 	ctx := context.Background()
 
-	runtime, guestModule, err := prepareRuntime(ctx, config.GuestPath)
+	guestBin, err := os.ReadFile(config.GuestPath)
+	if err != nil {
+		return nil, fmt.Errorf("wasm: error reading guest binary: %w", err)
```

Comment:
nit

```suggestion
		return nil, fmt.Errorf("wasm: error reading guest binary at %s: %w", config.GuestPath, err)
```

Reply:
nit of nit: I usually only make changes like this with unit tests to avoid duplicating the same message. Once we stabilize things we should add error tests so that we don't accidentally make things worse for not knowing how a message ends up.